pkg/kubeapiserver/default_storage_factory_builder.go

```diff
@@ -111,6 +111,7 @@ func (c *completedStorageFactoryConfig) New() (*serverstorage.DefaultStorageFact
 	storageFactory.AddCohabitatingResources(apps.Resource("daemonsets"), extensions.Resource("daemonsets"))
 	storageFactory.AddCohabitatingResources(apps.Resource("replicasets"), extensions.Resource("replicasets"))
 	storageFactory.AddCohabitatingResources(api.Resource("events"), events.Resource("events"))
+	storageFactory.AddCohabitatingResources(api.Resource("replicationcontrollers"), extensions.Resource("replicationcontrollers")) // to make scale subresources equivalent
```

Comment:
why wasn't that there before and which influence had that?

Reply:
extensions/v1beta1 replicationcontrollers *only* serve a `scale` subresource, no parent resource, so no storage asked about the prefix for an `extensions/v1beta1` replicationcontrollers resource previously. We now need to know what storage is modified by the scale subresource, and the storage factory is the thing we use to answer that question, so I needed to register this cross-group relationship